src/core/core.helpers.js

```diff
@@ -53,6 +53,16 @@ module.exports = function(Chart) {
 		}
 		return base;
 	};
+    // Adjust element index when min is defined
+	helpers.adjustIndex = function(element, chartConfig) {
+		var min = chartConfig.options.scales.xAxes[0].ticks.min;
+		var labels = chartConfig.data.labels;
+
+		if (!element._adjustedIndex && min) {
+			element._index -= labels.indexOf(min);
+			element._adjustedIndex = true;
+		}
+	};
```

Comment:
@simonbrunel, yes, this is a better place for the helper. I'll go ahead and update that